docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,117 @@
+# May 2024
+This roadmap is split into "themes" which roughly mirror our current priorities.
+
+## Data management
+**Current status**
+
+We have developed [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/), a simple, standard data folder 
+specification for (systems) neuroscience and [datashuttle](datashuttle.neuroinformatics.dev), to automate the 
+creation, validation and transfer of NeuroBlueprint projects. For more details on this specific project, see the 
+[datashuttle roadmap]().
+
+**Q4 2024**
+
+Standardised metadata support
+
+**Q2 2025**
+
+Support for customising data structure (e.g. specific mandatory elements or new file types)
+
+**Q4 2025**
+
+Export of NeuroBlueprint projects to [Neurodata Without Borders](https://www.nwb.org/) format.
+
+## Behavioural analysis
+**Current status**
+
+We are building [movement](https://movement.neuroinformatics.dev/) a Python toolbox for analysis of animal tracking 
+data. For more details on this specific project, see the [movement roadmap](https://movement.neuroinformatics.dev/community/roadmap.html).
+
+**Q3 2025**
+
+- Support all common post pose-estimation analysis tasks
+- Support for single experiments and complex multi-animal, multi-session datasets 
+(by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+**No current ETA**
+
+- A common interface to prevalent pose estimation tools, and other machine-learning based object detection methods
+- A common interface to methods for behavioural segmentation and classification
+
+## Extracellular electrophysiology analysis
+**Current status**
+
+We are currently building [spikewrap](https://github.com/neuroinformatics-unit/spikewrap) (built on top of 
```

Comment:
Maybe an aim can be added to mention in the short-term we will encourage SpikeInterface as a pipeline-builder, and provide tools, advice, facilitate communication on building pipelines through the 'ephys at the swc' site.

Reply:
TBH I don't think this should be here, as it's not really a product/deliverable/whatever you want to call it (although this is based on me not having looked at the "ephys at swc" site yet.

Reply:
Sure that makes sense, this is a more high-level NIU roadmap. For me it is useful to have a roadmap for ephys directly spefically to SWC users that I can point to, possibly on the 'Ephys at the SWC page'. I see you added comments related to this there so will discuss further there.

Reply:
Yes that works nicely @adamltyson, please see revision to this roadmap. I delete the ephys at SWC  roadmap and added a short section to the [index.md](https://github.com/neuroinformatics-unit/HowTo/blob/78624402c9935065af625d133a8cdc782a07d9fe/docs/source/ephys_at_swc/index.md?plain=1#L15) LMKWYT

Reply:
Looks nice. We can have another pass over everything once everyone else has reviewed.